docs/releasing/publishing.md

```diff
@@ -1,15 +1,30 @@
 # Publishing GOV.UK Frontend
 
-1. Make sure you have cards in the "Backlog" column to:
-  - Update the GOV.UK Design System to use the latest release
-  - Update the GOV.UK Prototype Kit to use the latest release
-  - Update the GOV.UK Frontend Docs to use the latest release
+## Steps you must complete before you release a new version of GOV.UK Frontend
 
 2. Checkout **master** and pull latest changes.
+1. Raise new issues in the team GitHub repositories to:
+- Create release notes for the new release, for example, [#1986](https://github.com/alphagov/govuk-frontend/issues/1986)
+- Create release notes for the new GOV.UK Prototype Kit v[VERSION NUMBER]   
+- Release v[VERSION NUMBER], for example, [#1987](https://github.com/alphagov/govuk-frontend/issues/1987)
+- Update the GOV.UK Design System to use the new release, for example, [#1347](https://github.com/alphagov/govuk-design-system/issues/1347)
+- Update the GOV.UK Frontend Docs to use the new release, for example, [#75](https://github.com/alphagov/govuk-frontend-docs/issues/75)
+- Update the GOV.UK Prototype Kit to use the new release, for example, [#923](https://github.com/alphagov/govuk-prototype-kit/issues/923)
+- Release GOV.UK Prototype Kit v[version number], for example, [#917](https://github.com/alphagov/govuk-prototype-kit/issues/917)
```

Comment:
In earlier items we use uppercase for `v[version number]` - can we make them consistent?

Reply:
I've actually updated the wording now so that we don't need to use placeholders, will ask @EoinShaughnessy to take a look too.